src/posit/connect/users.py

```diff
@@ -148,3 +148,8 @@ def update(self) -> User:
 
     def delete(self) -> None:
         raise NotImplementedError()
+
+    def count(self) -> int:
+        response: requests.Response = self.session.get(self.url, json={"page_size": 1})
```

Comment:
For my own edification: is this type hint necessary? Doesn't it (waves hands) know that `requests.Session.get()` returns `requests.Response`? And that `Response.json()` returns `dict`?

Reply:
No, it isn't necessary. Once I added the correct type declaration to `session` in the `__init__` method, this type declaration became redundant.

Your understanding is correct 😄